client/app/recordings/new/route.js

```diff
@@ -7,11 +7,62 @@ export default Route.extend({
     this.container = options.container;
   },
 
-  render() {
-    this.view = new View({
-      model: new Model(),
-      editing: true
-    });
-    this.container.show(this.view);
-  }
+  render(params) {
+    var that = this;
+    function renderPage(_model){
+      that.view = new View({
+        model: _model,
+      });
+      that.container.show(that.view);
+    }
+    function retrieveListAndRender(_model){
+      var listxhr = new XMLHttpRequest();
+      listxhr.onreadystatechange = function(){
+        var list = JSON.parse(listxhr.responseText)["data"]
```

Comment:
I generally think you should never use the `obj["objectKey"]` format when the `obj.objectKey` format is possible (when the key is known in advance). I think its better because you're clearly communicating an expectation for the format of the object, and in a way that is greppable. "data" here isn't some arbitrary string, it's something that your code is depending on being present in the response.